### ISSUE TITLE
Pause stream

### ISSUE THREAD

**Issue by GeekEdem** [mscdex/node-imap#629]
Hello. 
Is there any mechanism to pause stream/receiving letters before server complete working on received mail and after continue?
Thanks for your help.

**Comment by mscdex** [comment]
I don't understand what you're asking.

**Comment by GeekEdem** [comment]
Okay. Example from README.md

```
imap.once('ready', function() {
    openInbox(function(err, box) {
        if (err) throw err;
        var f = imap.seq.fetch('1:3', {
            bodies: 'HEADER.FIELDS (FROM TO SUBJECT DATE)',
            struct: true
        });
        f.on('message', function(msg, seqno) {
            console.log('Message #%d', seqno);
            ...
```

Can while I will doing some async operations on 'msg`, stop receiving new `'message'` events?

Something like `f.pause()` and `f.continue()`?
Thanks.

**Comment by mscdex** [comment]
No, there is currently no such functionality since the object returned by `fetch()` is just an `EventEmitter`. However, for message *bodies*, those are `Readable` streams which you *can* pause/resume.
